src/modules/toolbox/components/toolBar/ToolBar.js

```diff
@@ -73,6 +77,10 @@ export class ToolBar extends MvuElement {
 			return isOpen ? 'is-open' : '';
 		};
 
+		const getButtonClass = () => {
+			return isOpen ? 'showButton' : '';
+		};
+
```

Comment:
Found no test on presence or absence of .showButton class ( (initial AND runtime!)